Sources/VLCMedia.m

```diff
@@ -836,7 +840,7 @@ + (libvlc_meta_t)stringToMetaType:(NSString *)string
 #undef VLCStringToMeta
     }
     NSNumber * number = stringToMetaDictionary[string];
-    return (libvlc_meta_t) (number ? [number intValue] : -1);
```

Comment:
I think `-1` is not intuitive when the return type is `libvlc_meta_t`. `libvlc_meta_invalid` is later reused in `metadataForKey` and so on.